### DISCUSSION TITLE
Full Decorator inline html template parsing for angular

### DISCUSSION THREAD

**Discussion by jpike88** [biomejs/biome#628] [Ideas]
I know html formatting isn't built in yet but just fyi below

- When parsing a TS file, there may be an inline html template in the @Component decorator, would be good to cover that case
- New control flow syntax is about to roll into angular, prettier is just about to implement support for it. See link [here](https://medium.com/m/global-identity-2?redirectUrl=https%3A%2F%2Fblog.angular.io%2Fmeet-angulars-new-control-flow-a02c6eee7843)

**Comment by jpike88** [comment]
@ematipico @Conaclos I read https://biomejs.dev/blog/biome-wins-prettier-challenge/, congratulations. I use Prettier in our angular project, and it handles the above cases (which I would love to use biome for instead). is there any appetite to handle the above?

**Reply by ematipico** [reply]
I don't see this as an high priority in the core team, but if you or someone else wants to work on this, we are happy to help.

**Reply by jpike88** [reply]
With the new architecture in place, does this make things any more doable?

**Reply by jpike88** [reply]
(something like this would make it possible for angular devs to dump prettier/eslint and go full biome)
